contributors/design-proposals/raw-block-pv.md

```diff
@@ -0,0 +1,571 @@
+# Local Raw Block Consumption via Persistent Volume Source
```

Comment:
@erinboyd 
I mean the title of committed document would be "Raw block Consumption via Persistent Volume Source" or something like that. Just remove the "local".
I think the title of this PR is originally "Raw block". I'd prefer the title of this PR like "[Proposal] Raw block volume support" but this is nit.